app/indexers/work_indexer.rb

```diff
@@ -37,6 +37,9 @@ class WorkIndexer < Kithe::Indexer
     to_field ["text_no_boost_tesim", "medium_facet"], obj_extract("medium")
     to_field ["text_no_boost_tesim", "format_facet"], obj_extract("format"), transform(->(v) { v.titleize })
     to_field ["text_no_boost_tesim", "rights_facet"], obj_extract("rights") # URL id
+
+    to_field ["rights_facet"], obj_extract("rights"), transform(->(v) { "Copyright Free" if RightsTerm.copyright_free_filter_uris().include?(v) }) # human label
```

Comment:
As ruby style, we don't generally use parens (`()`) on method calls unless there are arguments. `RightsTerm.copyright_free_filter_uris.include?(v)` is preferable. 

We could probably make this more efficient by combining 39 and 41 into one line, but I don't know if it matters.  And I guess not easily if we _don't_ want to put `copyright free` in the text field.